src/map_events.c

```diff
@@ -31,63 +31,63 @@ static void MapEvents_ReadFromNarc(MapEvents *events, u32 mapno) {
     ReadWholeNarcMemberByIdPair(events->event_data, NARC_fielddata_eventdata_zone_event, bank);
 }
 
-void Field_InitMapObjectsFromZoneEventData(FieldSystem *fsys) {
-    u32 obj_count = fsys->mapEvents->num_object_events;
-    GF_ASSERT(fsys->mapEvents != NULL);
+void Field_InitMapObjectsFromZoneEventData(FieldSystem *fieldSystem) {
+    u32 obj_count = fieldSystem->mapEvents->num_object_events;
+    GF_ASSERT(fieldSystem->mapEvents != NULL);
     if (obj_count != 0) {
-        InitMapObjectsFromEventTemplates(fsys->mapObjectMan, fsys->location->mapId, obj_count, fsys->mapEvents->object_events);
+        InitMapObjectsFromEventTemplates(fieldSystem->mapObjectManager, fieldSystem->location->mapId, obj_count, fieldSystem->mapEvents->object_events);
     }
 }
 
-BG_EVENT *Field_GetBgEvents(FieldSystem *fsys) {
-    return fsys->mapEvents->bg_events;
+BG_EVENT *Field_GetBgEvents(FieldSystem *fieldSystem) {
+    return fieldSystem->mapEvents->bg_events;
 }
 
-u32 Field_GetNumBgEvents(const FieldSystem *fsys) {
-    return fsys->mapEvents->num_bg_events;
+u32 Field_GetNumBgEvents(const FieldSystem *fieldSystem) {
+    return fieldSystem->mapEvents->num_bg_events;
 }
 
-const WARP_EVENT *Field_GetWarpEventI(const FieldSystem *fsys, u32 warpno) {
-    MapEvents *events = fsys->mapEvents;
-    if (warpno >= fsys->mapEvents->num_warp_events) {
+const WARP_EVENT *Field_GetWarpEventI(const FieldSystem *fieldSystem, u32 warpno) {
+    MapEvents *events = fieldSystem->mapEvents;
+    if (warpno >= fieldSystem->mapEvents->num_warp_events) {
         return NULL;
     } else {
-        return &fsys->mapEvents->warp_events[warpno];
+        return &fieldSystem->mapEvents->warp_events[warpno];
     }
 }
 
-int Field_GetWarpEventAtXYPos(const FieldSystem *fsys, int x, int y) {
+int Field_GetWarpEventAtXYPos(const FieldSystem *fieldSystem, int x, int y) {
     int i;
 
-    for (i = 0; i < fsys->mapEvents->num_warp_events; i++) {
-        if (x == fsys->mapEvents->warp_events[i].x && y == fsys->mapEvents->warp_events[i].y) {
+    for (i = 0; i < fieldSystem->mapEvents->num_warp_events; i++) {
+        if (x == fieldSystem->mapEvents->warp_events[i].x && y == fieldSystem->mapEvents->warp_events[i].y) {
             return i;
         }
     }
 
     return -1;
 }
 
-u32 Field_GetNumCoordEvents(const FieldSystem *fsys) {
-    return fsys->mapEvents->num_coord_events;
+u32 Field_GetNumCoordEvents(const FieldSystem *fieldSystem) {
+    return fieldSystem->mapEvents->num_coord_events;
 }
 
-const COORD_EVENT *Field_GetCoordEvents(const FieldSystem *fsys) {
-    return fsys->mapEvents->coord_events;
+const COORD_EVENT *Field_GetCoordEvents(const FieldSystem *fieldSystem) {
```

Comment:
COORD_EVENT -> CoordEvent?

Reply:
Probably make an issue to just fix all of the GF_CASE struct names instead of doing it as you go along.